test/lib/get-group-tangle.test.js

```diff
@@ -77,34 +73,6 @@ test('get-group-tangle unit test', t => {
   })
 })
 
-test('get-group-tangle (cache)', t => {
-  const name = `get-group-tangle-cache-${Date.now()}`
-  const server = Server({ name })
-
-  let queryCalls = 0
-  server.backlinks.read.hook(function (read, args) {
-    queryCalls += 1
-
-    return read(...args)
-  })
-  server.tribes.create(null, (err, data) => {
-    if (err) throw err
-
-    // 1 for group tangle, 1 for members tangle
-    t.equal(queryCalls, 2, 'no cache for publishing of group/add-member, a backlink query was run')
-    const content = { type: 'memo', recps: [data.groupId] }
-
-    server.publish(content, (err, msg) => {
-      if (err) throw err
-
-      t.equal(queryCalls, 2, 'cache used for publishing next message')
-
-      server.close()
-      t.end()
-    })
-  })
-})
```

Comment:
Is there any cache on the group tangle still? 
it's kinda important to stop clobbering the DB I think - we have some bulk record creation from import supported, can make 1000 messages easily in one go. Cannot re-build tangle per message!

Reply:
we removed it also in tribes2, i think we said db2 is so good at caching anyway